edxval/tests/test_api.py

```diff
@@ -334,6 +334,45 @@ def test_get_videos_for_ids_duplicates(self):
         videos = list(api.get_videos_for_ids([edx_video_id, edx_video_id]))
         self.assertEqual(len(videos), 1)
 
+    def test_get_videos_for_ids_sort(self):
+        Video.objects.create(**constants.VIDEO_DICT_STAR)
+        fish_id = constants.VIDEO_DICT_FISH["edx_video_id"]
+        star_id = constants.VIDEO_DICT_STAR["edx_video_id"]
+
+        def check_sort(sort_field, sort_dir, expected_ids):
+            """
+            Assert that sorting by given field and direction returns videos in
+            the expected order
+            """
+            actual_videos = api.get_videos_for_ids(
+                expected_ids,
+                sort_field,
+                sort_dir
+            )
+            actual_ids = [video["edx_video_id"] for video in actual_videos]
+            self.assertEqual(actual_ids, expected_ids)
+
+        check_sort(
+            VideoSortField.client_video_id,
+            SortDirection.asc,
+            [fish_id, star_id]
+        )
+        check_sort(
+            VideoSortField.client_video_id,
+            SortDirection.desc,
+            [star_id, fish_id]
+        )
+        check_sort(
+            VideoSortField.duration,
+            SortDirection.asc,
+            [star_id, fish_id]
+        )
+        check_sort(
+            VideoSortField.duration,
+            SortDirection.desc,
+            [fish_id, star_id]
+        )
```

Comment:
Will do. As noted above, the videos have the same duration, so that case is covered.